photon-ml/src/main/scala/com/linkedin/photon/ml/function/glm/PointwiseLossFunction.scala

```diff
@@ -12,17 +12,28 @@
  * License for the specific language governing permissions and limitations
  * under the License.
  */
-package com.linkedin.photon.ml.function
+package com.linkedin.photon.ml.function.glm
 
 /**
- * The function to calculate l(z, y) with z = theta^T^x + offset for generalized linear model loss functions, where
- * theta is the coefficient, x is the feature vector, z is the margin, y is the label. It is the negative
- * of the log-likelihood for one data point, except for an irrelevant constant term.
+ * The function to calculate l(z, y) with:
+ *
+ *    z = theta^T^x + offset
```

Comment:
Minor: in some places we're using ```w``` for the weights, in others we're using ```theta```. Should be consistent.